exporter/opentelemetry-exporter-prometheus-remote-write/src/opentelemetry/exporter/prometheus_remote_write/__init__.py

```diff
@@ -142,50 +153,144 @@ def export(
     def shutdown(self) -> None:
         raise NotImplementedError()
 
-    def convert_to_timeseries(
+    def _convert_to_timeseries(
         self, export_records: Sequence[ExportRecord]
     ) -> Sequence[TimeSeries]:
-        raise NotImplementedError()
-
-    def convert_from_sum(self, sum_record: ExportRecord) -> TimeSeries:
-        raise NotImplementedError()
-
-    def convert_from_min_max_sum_count(
+        converter_map = {
+            MinMaxSumCountAggregator: self._convert_from_min_max_sum_count,
```

Comment:
We need to redefine this map on every call to convert? If you take the convert functions out of the exporter, you can just define it once at the top level.

Reply:
An alternative to extracting all the converter methods out of the class could be defining it once in the constructor as a class attribute.

Reply:
If I were to move the converter methods out of the class, would the map have to instantiated at file level in a separate converter.py file?